src/status_im/anon_metrics/interceptors.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.anon-metrics.interceptors
+  (:require [status-im.ethereum.json-rpc :as json-rpc]
+            [taoensso.timbre :as log]
+            [re-frame.interceptor :refer [->interceptor get-coeffect]]
+            [status-im.utils.platform :as platform]
+            [status-im.utils.build :as build]
+            [status-im.anon-metrics.transformers :as txf]))
+
+(defn catch-events-fn [context]
```

Comment:
`catch-events-fn` is a generic name, what do you think about `transform-and-log` or `transform-and-write-to-db`?